Makefile

```diff
@@ -164,7 +164,7 @@ build/config.BUILT:
 build: build/llvm.BUILT build/wasi-libc.BUILT build/compiler-rt.BUILT build/libcxxabi.BUILT build/libcxx.BUILT build/config.BUILT
 
 strip: build/llvm.BUILT
-	cd $(PREFIX)/bin; strip clang-* clang++-* lld llvm-ar
+	./strip_symbols.sh $(PREFIX)/bin
```

Comment:
Given the existence of this script I wonder if we should just pass `$(PREFIX)`?   Maybe there are ELF files in other directories and we want to strip those too, no?

Reply:
I'm not opposed to doing that; I just was concerned about stripping symbols where I shouldn't. Are we sure it's ok to do it on the entire `$(PREFIX)`?

Reply:
This is the approach the debian packages normally take IIRC.. strip *all* ELF files by default.  I see no reason not to do it here... but I could be missing something.